sky/sdk/lib/widgets/variable_height_scrollable.dart

```diff
@@ -14,18 +12,28 @@ class VariableHeightScrollable extends Scrollable {
   VariableHeightScrollable({
     String key,
     this.builder,
-    this.token
+    this.token,
+    this.layoutState
   }) : super(key: key);
 
   IndexedBuilder builder;
   Object token;
+  BlockViewportLayoutState layoutState;
   bool _contentsChanged = true;
 
+  void initState() {
+    if (layoutState == null)
```

Comment:
We should decide if we want it to be passed into the constructor or if we're going to manage it. We shouldn't do both.